packages/react/src/components/List/List.stories.tsx

```diff
@@ -1,26 +1,51 @@
 import React from 'react';
-import type { Meta, StoryFn } from '@storybook/react';
+import type { Meta } from '@storybook/react';
 
 import type { ListProps } from '.';
-import { List, ListItem } from '.';
+import { List } from '.';
 
 const meta: Meta<typeof List> = {
-  title: 'Altinn/List',
+  title: 'Felles/List',
   component: List,
   args: {
     // Just to make the default option pre-selected i <Controls />
-    borderStyle: 'solid',
+    as: 'ul',
+    size: 'medium',
   },
```

Comment:
Suggest only applying this to the `Preview` story like we do with our other component stories as to not confuse our users when using "show code" on the other stories as these are default values.